layers/+source-control/git/README.org

```diff
@@ -59,6 +64,20 @@ file.
 Of course if your OS does not ship with git (!) you'll have to install it
 on your machine. You can download it from the [[http://git-scm.com/downloads][download page]].
 
+*** Basic Git configuration
+Define your git identity using the following commands in a terminal window
+replacing username and name@domain.tld with your own values
+
+#+begin_src shell
+  git config --global user.name "username"
+  git config --global user.email "name@domain.tld"
+#+end_src
+
+*** Using SSH URLs for remote repositories
+When using an SSH URL for remote repository access, [[https://help.github.com/articles/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent/][generate an SSH key]] and
+add it to your GitHub // GitLab account to remove the need to provide
+login details for each request from Magit to the remote repository service.
```

Comment:
I feel this is a little bit out of place. At this point, the reader may just want to use Git and may not even know what GitHub/GitLab is

Reply:
Even if some readers want only to use Git, this information should be provided for others who want to set up their system for Magit to connect securely to remote repositories.

Reply:
Hm, I'm not sure. 

1. This is layer docs, not git docs. You can use git with and without SSH. But whether you do or not is not related to magit or other packages in this layer. I think this part is generic git config but has nothing to do with the layer.
2. git (with or without SSH) is not limited to GitHub and GitLab, so why mention these two here? Mentioning them below is fine when talking about forge because forge is explicitly built for those.

But I don't have a strong opinion on this.

Reply:
I've separated the forge specific details.

I dont get the use case for using git without a remote repository service.  Seems a very niche use case.

One of the biggest problems I've had / still have occasionally with Spacemacs is installing something and then having to spend a day (or a week in Magit Forge's case) trying to figure out how to configure it properly (and securely).  This is why I wrote Practicalli Spacemacs as the layer docs are so limited.

I appreciate that the Spacemacs layer docs are supposed to be limited.  But with that being the case then none of this PR should be accepted.

Thank you.